open_lm/norms.py

```diff
@@ -69,6 +69,41 @@ def extra_repr(self) -> str:
         )
 
 
+class LPLayerNorm(torch.nn.LayerNorm):
```

Comment:
if you extend the `LayerNorm` class above on L12 which is basically `torch.nn.LayerNorm` we can also expdriment with LPLayerNorm without biases

Reply:
ok done! (btw is there a reason we don't subclass LayerNorm from torch.nn.LayerNorm?)